ui/yarn.lock

```diff
@@ -7380,7 +7381,7 @@ ember-source@~3.28.10:
     semver "^7.3.4"
     silent-error "^1.1.1"
 
-ember-svg-jar@^2.2.3, ember-svg-jar@^2.3.4:
+ember-svg-jar@^2.2.3:
```

Comment:
**[observation, non-blocking]**

Ah, looks like we can’t retire ember-svg-jar entirely because @hashicorp/pds-ember depends on it. Not a big deal, just noting for posterity.